virt-who-config.5

```diff
@@ -116,6 +116,12 @@ Alternative to the rhsm_proxy_password option; encrypted password generated by t
 \fBrhsm_no_proxy\fR
 Optional proxy settings for the Subscription Asset Manager or Satellite 6 server to use in place of the no proxy filter defined in the system's rhsm.conf.
 .TP
+\fBrhsm_certificate\fR
```

Comment:
This is not used in virt-who. It is a setting in subscription-manager, but virt-who has never passed it from its own configuration.

Reply:
when I tried, I saw that really doing nothing, that explain the root cause, hahaha

sorry, but I only can to read a few code lines and try to test if my interpretation of the readed code works as my needs

thanks for the aclaration